driver-core/src/main/com/mongodb/internal/operation/AggregateOperationImpl.java

```diff
@@ -201,7 +201,7 @@ private CommandCreator getCommandCreator(final SessionContext sessionContext) {
         return new CommandCreator() {
             @Override
             public BsonDocument create(final ServerDescription serverDescription, final ConnectionDescription connectionDescription) {
-                validateReadConcernAndCollation(connectionDescription, sessionContext.getReadConcern(), collation);
+                validateFindOptions(connectionDescription, sessionContext.getReadConcern(), collation, allowDiskUse);
```

Comment:
Can I still use allowDiskUse in the aggregation but not in the find?  I think that needs a to be supported and tested.